protos/bastionlab_conversion.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package bastionlab_conversion;
+import "bastionlab.proto";
+
+message ToTensor {
+    string identifier = 1;
+}
+
+message ToDataFrame {
+    repeated string inputs_col_names = 1;
+    string labels_col_name = 2;
+    string inputs_conv_fn = 3;
+    string labels_conv_fn = 4;
+    string identifier = 5;
+}
+
+service ConversionService {
+    rpc ConvToTensor(ToTensor) returns (bastionlab.Reference) {}
```

Comment:
(i am reviewing with lucas)
Now that we have bastionlab.Reference, I think we can move this back in the polars protobuf directly, what do you think? We won't have import problems anymore, do you think that would work?
Sorry if this means changing things a lot :sweat_smile: